src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx

```diff
@@ -20,14 +23,17 @@ export const secondSidebarBlock = {
   content: (
     <>
       <p>
-        Please wait at least 3 business days (Monday-Friday) before reporting
-        non-receipt of a payment as check tracing cannot be initiated before
-        that period. To report non-receipt after 3 business days, call
-        <a href="tel:8008271000" aria-label="1. 800. 827. 1000.">
-          800-827-1000
-        </a>{' '}
-        with your Social Security Number or VA Claim Number, your address, and
-        (for direct deposit payments) your account information.
+        Please wait 3 business days (Monday-Friday) before contacting us to
```

Comment:
Screen readers will read out "dash", so it's better to use "Monday through Friday" here (we should double check this with our a11y gurus); but if your design *must* use a dash, then an [en-dash](https://design.va.gov/content-style-guide/punctuation#em-and-en-dashes-hyphens) (`&ndash;`) should likely be used here.

Reply:
Great callout, updated